catalog/spatial/docs/src/main/resources/_contents/mancontents.adoc

```diff
@@ -38,42 +37,42 @@ These Installation steps are the same whether {branding} was installed from a di
+If the ${branding} application is deployed on the ${branding} Kernel in a custom installation (or the application has been upgraded previously), i.e., its KAR file is in the `<INSTALL_DIRECTORY>/deploy` directory, uninstall it by deleting this KAR file.
 
-Otherwise, if the {branding} application is running as part of the {branding} distribution zip, it is uninstalled the first time and only the first time using the `features:removeurl` command:
+Otherwise, if the ${branding} application is running as part of the ${branding} distribution zip, it is uninstalled the first time and only the first time using the `feature:repo-remove` command:
 
-.Uninstall {branding} application from {branding} distribution
+.Uninstall ${branding} application from ${branding} distribution
 ----
-features:removeurl -u <Application's feature repository URL>
+feature:repo-remove -u <Application's feature repository URL>
 
-Example:   features:removeurl -u mvn:org.codice.ddf.spatial/spatial-app/2.5.0/xml/features
+Example:   feature:repo-remove -u mvn:org.codice.ddf.spatial/spatial-app/2.5.0/xml/features
 ----
 
-The uninstall of the application can be verified by the absence of any of the {branding} application's features in the `features:list` command output.
+The uninstall of the application can be verified by the absence of any of the ${branding} application's features in the `feature:list` command output.
 
 [NOTE]
 ====
 The repository URLs for installed applications can be obtained by entering:
-      `features:listrepositories -u`
+      `feature:repo-list -u`
 ====
 
 ==== Reverting the Uninstall
 
-If the uninstall of the {branding} application needs to be reverted, this is accomplished by either:
+If the uninstall of the ${branding} application needs to be reverted, this is accomplished by either:
 
 * copying the application's KAR file previously in the `<INSTALL_DIRECTORY>/deploy` directory, OR
-* adding the application's feature repository back into {branding} and installing its main feature, which typically is of the form `<applicationName>-app`, e.g., `catalog-app`.
+* adding the application's feature repository back into ${branding} and installing its main feature, which typically is of the form `<applicationName>-app`, e.g., `catalog-app`.
 
-.Reverting {branding} application's uninstall
+.Reverting ${branding} application's uninstall
 ----
-features:addurl <Application's feature repository URL>
-features:install <Application's main feature>
+feature:repo-add <Application's feature repository URL>
+feature:install <Application's main feature>
 ----
 
 .Example
 ----
-ddf@local>features:addurl mvn:ddf.catalog/catalog-app/2.3.0/xml/features
-ddf@local>features:install catalog-app
+ddf@local>feature:repo-add mvn:ddf.catalog/catalog-app/2.3.0/xml/features
```

Comment:
Should we maybe push the sample version number up to more closely reflect the present?